### DISCUSSION TITLE
Hexo添加全局音乐播放-APlayer(包含位置修改) - 没有秃头基因的燊的博客

### DISCUSSION THREAD

**Discussion by giscus** [yzs020220/giscus-repo#7] [Announcements]
# Hexo添加全局音乐播放-APlayer(包含位置修改) - 没有秃头基因的燊的博客

使用APlayer+MetingJS方便快捷地为你的博客添加歌单，qq音乐不能外链，建议网易

https://yzs020220.github.io/posts/47641/

**Comment by HCY-ASLEEP** [comment]
💪
